Running mvn test fails

### Expected Behaviour

`mvn test` builds the project succesfully.

### Actual Behaviour

```
[ERROR] Failed to execute goal org.apache.jackrabbit:filevault-package-maven-plugin:1.0.1:analyze-classes (default-analyze-classes) on project test.ui.apps: Error while analysing imports: /home/robert/sources/test/core/target/classes (Is a directory) -> [Help 1]
```

### Reproduce Scenario (including but not limited to)

Run `mvn test`

--- a/src/main/archetype/pom.xml
+++ b/src/main/archetype/pom.xml
@@ -202,7 +202,7 @@
                 <plugin>
                     <groupId>org.apache.jackrabbit</groupId>
                     <artifactId>filevault-package-maven-plugin</artifactId>
-                    <version>1.0.1</version>
+                    <version>1.0.3</version>
                     <configuration>
                         <filterSource>src/main/content/META-INF/vault/filter.xml</filterSource>
                     </configuration>

